docs/docs/30-how-to-guides/14-working-with-stages.md

```diff
@@ -0,0 +1,548 @@
+    status:
+      finishedAt: "2024-07-15T22:13:25Z"
+      freight:
+        commits:
+        - healthCheckCommit: 111eaf55aa41f21bb9bb707ba1baa748b83ec51e
+          id: 961cfaedbc53aacdb65110028839a2c1c281290d
+          repoURL: https://github.com/example/kargo-demo.git
+        name: 666209fd9755a1e48bec6b27f5f447747410dd9e
+        origin:
+          kind: Warehouse
+          name: kargo-demo
+      freightCollection:
+        id: 101bca5b0e18ca7913978a1da956308d2544f741
+        items:
+          Warehouse/kargo-demo:
+            commits:
+            - healthCheckCommit: 111eaf55aa41f21bb9bb707ba1baa748b83ec51e
+              id: 961cfaedbc53aacdb65110028839a2c1c281290d
+              repoURL: https://github.com/example/kargo-demo.git
+            name: 666209fd9755a1e48bec6b27f5f447747410dd9e
+            origin:
+              kind: Warehouse
+              name: kargo-demo
+        verificationHistory:
+        - analysisRun:
+            name: test.01j2w7aknhf3j7jteyqs72hnbg.101bca5
+            namespace: kargo-demo-09
+            phase: ""
+          id: 5535a484-bbd0-4f12-8cf4-be2c8e0041c9
+          phase: Pending
+          startTime: "2024-07-15T22:13:34Z"
+      phase: Succeeded
+  observedGeneration: 1
+  phase: Steady
+```
+
+## Interacting with Stages
+
+Kargo provides tools to manage Stages using either the Kargo Dashboard or the
+Kargo CLI. This section explains how to handle Stages effectively through both interfaces.
+
+### Creating a Stage
+
+<Tabs groupId="create-stage">
+<TabItem value="ui" label="Using the Kargo Dashboard" default>
+
+1. Navigate to your Project dashboard and locate the action menu in the right corner of the pipeline:
+
+   ![create-stage](../../static/img/create-stage.png)
+
+2. Click the magic wand icon to open the dropdown, then select <Hlt>Create Stage</Hlt>.
+   A form will appear to input details for the new `Stage`:
+
+   ![create-stage](../../static/img/create-stage-2.png)
+
+3. Complete the form with the necessary details and submit it.
+   The new `Stage` will be added to the pipeline, connected to other
+   `Stage`s based on your configuration:
+
+   ![create-stage](../../static/img/create-stage-3.png)
+</TabItem>
+
+<TabItem value="cli" label="Using the Kargo CLI">
+
+1. Define the `Stage` in a YAML file, for example:
+
+```yaml
+apiVersion: kargo.akuity.io/v1alpha1
+kind: Stage
+metadata:
+  name: <stage-name>
+  namespace: <project-name>
+spec:
+  ### Add your Stage specifications here
+```
+
+2. Save the file and run:
+
+```shell
+kargo create -f <stage-filename>
+```
+
+3. Verify the creation by listing the `Stage`:
+
+```shell
+kargo get stage <stage-name> --project <project-name>
+```
+
+</TabItem>
+</Tabs>
+
+### Deleting a Stage
+
+<Tabs groupId="delete-stage">
+<TabItem value="ui" label="Using the Kargo Dashboard" default>
+
+1. Select the `Stage` you want to remove.
+2. Click <Hlt>Delete</Hlt> in the top-right corner of the pop-up window:
```

Comment:
Changing this only because we used "upper right" instead of "top right" somewhere else.

```suggestion
2. Click <Hlt>Delete</Hlt> in the upper right corner of the pop-up window:
```